apps/prairielearn/src/pages/instructorIssues/instructorIssues.html.ts

```diff
@@ -319,9 +319,8 @@ function IssueRow({
 
 function getFormattedMessage(issue: Issue) {
   if (!issue.student_message) return html`&mdash;`;
-
-  const message = joinHtml(issue.student_message.split(/\r?\n|\r/), html`<br />`);
-  return issue.manually_reported ? html`"${message}"` : message;
```

Comment:
I believe this is basically equivalent to just using `white-space: pre-wrap;`, which I switched to in this PR.